### ISSUE TITLE
cohortDiagnostics in plp skeleton?

### ISSUE THREAD

**Issue by ted9219** [OHDSI/SkeletonPredictionStudy#7]
How about adding the cohortDiagnostics function in patient level prediction skeleton? 
I already have experience with that, so I can share and put the PR.

**Comment by jreps** [comment]
I agree :). We are thinking about ways to combine the different study type jsons (e.g., cohortDiagnostic json, characterization json and prediction json) into one study package - that should then make it easy to run the diagnostic before the prediction.  Let us know if you have any Ideas for this
